_targets.R

```diff
@@ -101,5 +101,27 @@ list(
                   file_out = "flow_cartogram.png",
                   width = 16),
     format = "file"
+  ),
+  
+  # Flow timeseries nationally - instagram 
+  tar_target(
+    flow_national_instagram_png,
+    national_ig(file_svg = "flow_national_ig.png", 
+                  plot_left = plot_nat, 
+                  date_start,
+                  width = 1080, height = 1080, color_bknd),
+    format = "file"
+  ),
+  
+  # Flow timeseries for states - instagram
+  tar_target(
+    flow_cartogram_instagram_png,
+    cartogram_ig(file_svg = "flow_cartogram_ig.png", 
+                 plot_left = plot_nat,
+                plot_right = plot_cart, 
+                date_start,
+                width = 1080, height = 1080, color_bknd),
+    format = "file"
```

Comment:
To parallel the steps in the twitter version, you should slightly modify this target and add another after:
```
 # Flow timeseries for states - instagram
  tar_target(
    flow_cartogram_instagram_svg,
    cartogram_ig(file_svg = "flow_cartogram_ig.svg", 
                 plot_left = plot_nat,
                plot_right = plot_cart, 
                date_start,
                width = 1080, height = 1080, color_bknd),
    format = "file"
  ),
  # Remove facet clipping and save as png
  tar_target(
    flow_cartogram_instagram_png,
    rm_facet_clip(svg_in = flow_cartogram_instagram_svg, 
                  file_out = "flow_cartogram_ig.png",
                  width = 16),
    format = "file"
  )
```

The `rm_facet_clip` function is taking the svg output and removing clipping boundaries that are around the drop shadows behind each tile. Without this, there is a strict border that cuts the shadow off abruptly. It's doing this by first saving the tile map as an svg. Then it is hacking into the svg code to delete the `clipPath` elements that are responsible for that.  I'm not sure this will matter much when the tile map is at a much smaller size. I suggest evaluating what it looks like with this extra step or not to decide if it is needed here.

Reply:
Ahh I see now, I do like them without the strict borders ![flow_cartogram_ig](https://user-images.githubusercontent.com/69014273/209014532-ac32c69d-9521-486f-b417-bdb85e12a938.png)

Reply:
Yeah I agree, the shadow is a little too visually dominant. That could be adjusted in the cartogram plotting function by adjusting [`sigma` or `offset` in `with_shadow`](https://github.com/USGS-VIZLAB/flow-tiles/blob/b5fb678214e12fa1f20aca5a409ad0c0c1cc2980/src/plot_cartogram.R#L52-L59). You'd want to rework the function/pipeline to make that changeable between platforms.